.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+- 1.11
```

Comment:
recommended to have this with quotes "1.11" just in case you downgrade to 1.10 then it would be 1.1